### PR TITLE
Allow mods to ban+unban, remove archived from docs

### LINKED ISSUE #155: Change permissions on commands
Commands should have their permissions changed to the following:

_Mods and above_
- [x] `cc!ban`
- [x] `cc!unban`

**Aside**

- [x] `cc!help` should not display any archived commands

### PR DESCRIPTION
**Pull request by Victoria-DR**
## What issue is this solving?

Closes #155 

### Description

- Allow mods to ban and unban users.
- Remove the four archived Discourse-related commands from cc!help and README.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? NO
- Any new dependencies to install? NO
- Any special requirements to test? Moderator role.
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

**Review by Vic-ST — APPROVED** [review]
LGTM!

File: commands/utility/help.js [hunk 1]
```diff
@@ -168,10 +140,7 @@ module.exports = {
           'Use `cc!help [command]` to get more information about a particular command.'
         )
         .setColor('DARK_NAVY')
-        .addField(
-          'Admin Only',
-          'cc!createroles, cc!deleteroles, cc!ban, cc!unban, cc!clearinfractions'
-        )
+        .addField('Admin Only', 'cc!ban, cc!unban, cc!clearinfractions')
```
Comment (aedwardg) [inline commands/utility/help.js:143]: Ban and unban shouldn't be listed as Admin-only anymore after this change.

**Review by aedwardg — CHANGES_REQUESTED** [review]
@Victoria-DR, 

A couple of tiny changes and then this will be ready to go.

**Review by aedwardg — APPROVED** [review]
Thanks @Victoria-DR,

This looks ready to go.

**Pull request merged**